create-tokens.py

```diff
@@ -108,15 +113,15 @@ def create_tokens(ds, **kwargs):
                             AND active = true 
                             AND approved = true 
                             AND token_id IS NULL 
-                            LIMIT 3000
+                            LIMIT .format(mintLimit)
```

Comment:
not sure about this one though... syntax might be wrong

Reply:
We need to test it fully, I remember this syntax for variables:
f""" some {variable}"""

@CareyAltgilbers can you also help on this?

Reply:
@AsarePhilip there it is

Reply:
@dadiorchen Yes, I'll be happy to take a look at it.

Reply:
Hello, 

It looks like on line 61 kwards is a typo and should be kwargs.                          

In the SQL statement on line 116 we should be able to change LIMIT .format(mintLimit) to Limit {} and then on line 117 change that to """.format(entityId, mintLimit))

That gives us two unnamed variables in this SQL statement, one on line 110 in a function call and another in line 116.

I ran the code in a test script and did not get any errors, if you have some test data I can use for the entityId and mintLimit I would test the test script again.

cursor.execute("""
    select id, uuid, token_id from trees
    where
        planter_id IN (
            select id from planter 
            where 
                organization_id IN ( 
                    select entity_id from getEntityRelationshipChildren({}) 
                ) 
            ) 
             AND active = true 
             AND approved = true 
             AND token_id IS NULL 
             LIMIT {}
             """.format(entityId, mintLimit))

Reply:
changed it. Should be fine now. see new change request